va/va_test.go

```diff
@@ -316,10 +317,20 @@ func TestHTTP(t *testing.T) {
 		t.Fatalf("Domain name is invalid.")
 	}
 	test.AssertEquals(t, prob.Type, probs.UnknownHostProblem)
+}
+
+func TestHTTPTimeout(t *testing.T) {
+	chall := core.HTTPChallenge01()
+	setChallengeToken(&chall, expectedToken)
+
+	hs := httpSrv(t, chall.Token)
+	// TODO(#1989): close hs
```

Comment:
This is no longer relevant, we can `defer hs.Close()` now, not sure why the other `TODO`s in here weren't closed in #2560.